benchmarks/metadata.py

```diff
@@ -9,13 +10,21 @@ def extract_name(definition):
     parser = MetadataParser(definition)
     return parser.name
 
+def extract_device(definition):
+    parser = MetadataParser(definition)
+    return parser.device
+
+def __format_device_name__(device):
+    return re.sub('mn-', '', device)
```

Comment:
I'm not sure this is a good idea to tamper with arbitrary device name strings from the lab? Would it make any harm to leave 'as is'?